web/packages/teleterm/package.json

```diff
@@ -2,23 +2,21 @@
   "name": "@gravitational/teleterm",
   "version": "1.0.0-dev",
   "description": "Teleport Connect",
-  "main": "build/app/dist/main/main.js",
+  "main": "build/app/main/index.js",
   "author": {
     "name": "Gravitational, Inc.",
     "email": "support@goteleport.com",
     "url": "https://goteleport.com"
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "yarn build-native-deps && webpack serve --config webpack.renderer.dev.config.js --progress",
-    "start-main": "webpack build --config webpack.main.config.js --mode=development --progress --watch",
+    "start": "yarn build-native-deps && yarn start-skip-native-deps",
+    "start-skip-native-deps": "electron-vite dev",
```

Comment:
Regarding hot reloading of main process and preload: I just tried it and it seems to work okay. You just do `yarn start-term -w` and electron-vite automatically restarts the app whenever the main process or the preload code changes. It restarts on changes to the shared process and the agent cleanup daemon too.

Restarting the app steals focus, that's one minus.

@gzdunek Can you think of a situation where we'd want to rebuild the main process but not restart the app immediately?

I feel like the `-w` option works well enough, we could work with it for a week or two and then see what could be adjusted. Besides, who cares about hot reloading when you can just rerun `yarn start-term` in a few seconds.

Reply:
>@gzdunek Can you think of a situation where we'd want to rebuild the main process but not restart the app immediately?

Nothing comes to my mind.

>I feel like the -w option works well enough, we could work with it for a week or two and then see what could be adjusted. Besides, who cares about hot reloading when you can just rerun yarn start-term in a few seconds.

I agree.

Reply:
I see that reloading doesn't work for the renderer:
```
MAIN [Main] warn: Navigation to https://localhost:8080/ blocked by 'will-navigate'
```
I will add an exception in dev mode for localhost:8080.

Reply:
The previous mechanism we had with F6 restarting the app was useful in situations where you modified just tsh and wanted to restart the app. There's probably a way to set up a rollup watcher in electron-vite that's going to restart the app on changes to `build/tsh`, but that's not essential for now.